### DISCUSSION TITLE
Nooted Kext : Gave me a blank screen after apple logo.

### DISCUSSION THREAD

**Discussion by Orinmoon** [ChefKissInc/NootedRed#88] [General]
i get the blank screen when i use DP display and when i switch to HDMI dysplay it worked.
- ROG Strix B550-I GAMING
- AMD Ryzen 5 5600G
- LATEST BIOS
- LOAD DEFAULT BIOS SETTING 

Reset NVram 3 times or more when u change SMbios. 
[EFI.zip](https://github.com/NootInc/NootedRed/files/11711769/EFI.zip)

**Comment by Orinmoon** [comment]
I Used MONTEREY for this project.

**Comment by Orinmoon** [comment]
Use SMBIOS Macmini8,1

**Comment by Orinmoon** [comment]
The efi is using MacPro7,1 works once in a while

**Comment by VisualEhrmanntraut** [comment]
Read the README about SMBIOS selection
